README.md

```diff
@@ -132,7 +132,7 @@ After getting familiar with the basic scenario, you may replace the test caniste
 2. Start a local replica instance:
     ```bash
    SNS_TESTING_INSTANCE=$(
-       docker run -p 8080:8080 -v "`pwd`":/dapp -d martin2718/sns-testing:latest dfx start --clean
+       docker run -p 8080:8080 -v "`pwd`":/dapp -d dfinity/sns-testing:main dfx start --clean
```

Comment:
What's the meaning of the tag `:main`? Typically, images have `:latest`, `:<version-number>`, or no tags at all (if they were built locally).

Reply:
This is the tag we'll get using the default [template](https://docs.docker.com/build/ci/github-actions/manage-tags-labels/) for managing tags and labels with Github actions (also see [here](https://github.com/docker/metadata-action#basic))

Reply:
Sounds reasonable